.env.example

```diff
@@ -18,6 +18,7 @@ BYOC_BBOX_EPSG3857_MAXX=
 BYOC_BBOX_EPSG3857_MAXY=
 S5PL2_LAYER_ID=
 LANDSAT8_LAYER_ID=
+LANDSAT8_NDVI_LAYER_ID=
```

Comment:
Don't forget to update .env file in out docs repo.

Reply:
Thanks, updated.